src/main/java/io/github/minecraftcursedlegacy/impl/networking/PluginChannelRegistryImpl.java

```diff
@@ -16,6 +16,6 @@ public static void registerPluginChannel(PluginChannel channel) {
 	}
 
 	protected static void handlePacket(PacketHandler arg, PluginMessagePacket packet) {
-		channelMap.get(packet.channel).onRecieve(arg, packet.data);
+		channelMap.get(packet.channel).onReceive(arg, packet.data);
 	}
 }
```

Comment:
No trailing newline. fix or remove.

Reply:
you mean at the end of the class. thocc

Reply:
Yeah, if you don't have a trailing \n on the end of the file git will act like that line's been changed if you add anything after it. Not really an issue, just makes diffs a byte or so longer.